src/AWS/Orleans.Reminders.DynamoDB/SiloHostBuilderReminderExtensions.cs

```diff
@@ -43,7 +43,7 @@ public static ISiloHostBuilder UseDynamoDBReminderService(this ISiloHostBuilder
         public static IServiceCollection UseDynamoDBReminderService(this IServiceCollection services, Action<DynamoDBReminderStorageOptions> configure)
         {
             services.AddSingleton<IReminderTable, DynamoDBReminderTable>();
-            services.Configure(configure);
+            services.Configure<DynamoDBReminderStorageOptions>(configure);
```

Comment:
This is not required, is it? Is there a reason why being explicit is better here? Not pushing back, just commenting, as I don't necessarily find it better (unless it was required to compile to the right thing)